index.html

```diff
@@ -253,9 +253,22 @@ <h4>Skills</h4>
 
 
         </div>
+           <section id="contact-me">
+            <h3 class="contatct-head">Contact me</h3>
+            <p class="contact-desc">If you have an application you <br> are interested in developing, a <br> feature that you
+                need built or a <br> project that needs coding. <br>
+                I'd love to help with it!</p>
+            <form action="https://formspree.io/f/xlekpvwq" method="post">
+                <input type="text" id="fullname" name="fullname" maxlength="30" placeholder="Lorem Ipsum">
```

Comment:
- [OPTIONAL] Please consider adding a meaningful placeholder to the name input such as `Enter your name` or simply `Full Name`, so this will be delivered with the power of a screen reader to the user that has disabilities, its our job to facilitate the use of the web to all people 🦸🏼‍♂️🦸🏼‍♀️